packages/tracing/src/browser/web-vitals/getLCP.ts

```diff
@@ -22,6 +22,17 @@ import { onHidden } from './lib/onHidden';
 import { whenInput } from './lib/whenInput';
 import { ReportHandler } from './types';
 
+// https://wicg.github.io/largest-contentful-paint/#sec-largest-contentful-paint-interface
+export interface LargestContentfulPaint extends PerformanceEntry {
+  renderTime: DOMHighResTimeStamp;
+  loadTime: DOMHighResTimeStamp;
+  size: number;
+  id: string;
+  url: string;
+  element?: Element;
+  toJSON(): Record<string, unknown>;
```

Comment:
Why do we use `unknown` here?

Reply:
I missed that. `toJSON` cannot shouldnt do anything else than `Record<string, string>`

Reply:
I updated this in https://github.com/getsentry/sentry-javascript/pull/3427/commits/5f42677aede1828ff1e5166fa4513c8ce10dc406